frontend/src/components/camp/CampCategories.vue

```diff
@@ -6,39 +6,39 @@ Displays all periods of a single camp and allows to edit them & create new ones
   <content-group>
     <slot name="title">
       <div class="ec-content-group__title py-1 subtitle-1">
-        {{ $tc('components.camp.campActivityCategories.title') }}
-        <dialog-activity-category-create :camp="camp()">
+        {{ $tc('components.camp.CampCategories.title') }}
```

Comment:
Übersetzungsschlüssel-Segmente sollen mit Kleinbuchstaben beginnen: https://github.com/ecamp/ecamp3/wiki/Localization-and-translations#rules-for-working-with-translations, basiert auf der Konvention von Vue i18n.